scripts/lint.ts

```diff
@@ -0,0 +1,26 @@
+import chalk from 'chalk'
+import { generateVSCodeLink, hasAnyArrayWithValues } from './helpers.js'
+import lintContent from './lintContent.js'
+
+async function lintMdFiles(path: string) {
+  const result = await lintContent([path])
+  const keys = Object.keys(result)
+  const values = Object.values(result)
+
+  if (hasAnyArrayWithValues(values)) {
```

Comment:
Maybe we can flatten the nested if statements here with inversion:
`if (!hasAnyArrayWithValues(values)) return false`

Reply:
for sure. will update!